src/calculator.js

```diff
@@ -1,48 +1,37 @@
-exports._check = () => {
+exports._check = (x,y) => {
   // DRY up the codebase with this function
   // First, move the duplicate error checking code here
   // Then, invoke this function inside each of the others
   // HINT: you can invoke this function with exports._check()
-};
 
-exports.add = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
-  return x + y;
-};
+    if (typeof x !== 'number') {
+      throw new TypeError(`${x} is not a number`);
+    }
+    if (typeof y !== 'number') {
+      throw new TypeError(`${y} is not a number`);
+    }
+  };
+  
+  exports.add = (x,y) => {
+    exports._check(x,y);
+    return x + y;
+  };
 
-exports.subtract = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
-  return x - y;
-};
 
-exports.multiply = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
-  return x * y;
-};
 
-exports.divide = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
-  return x / y;
-};
+  exports.subtract = (x, y) => {
+    exports._check(x,y);
+    return x - y;
+  };
+
+  exports.multiply = (x, y) => {
+    exports._check(x,y);
```

Comment:
A space is required after &#39;,&#39;.

###### See http://eslint.org/docs/rules/comma-spacing for details about this rule